### PR TITLE
CS Falcon: fix wrong code

### PR DESCRIPTION
**Pull request by ilappe**
## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/XSUP-41609

## Description
wrong `for` loop when the `resources` are `None`

**new test before fix**:
image

**after**:
image

### CODE REVIEW

File: Packs/CrowdStrikeFalcon/ReleaseNotes/2_0_16.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### CrowdStrike Falcon
+
+- Fixed an issue where the command **cs-falcon-search-device** failed when used with **extended_data=yes**.
```
Comment (ShahafBenYakir) [inline Packs/CrowdStrikeFalcon/ReleaseNotes/2_0_16.md:6]:
```suggestion
- Fixed an issue where the command **cs-falcon-search-device** failed when used with ***extended_data=yes***.
```

File: Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon_test.py [hunk 2]
```diff
@@ -7523,3 +7523,21 @@ def test_fix_time_field():
     assert detection_1['created_timestamp'] == '2023-04-20T11:13:10.424647Z'
     assert detection_2['created_timestamp'] == '2023-04-20T11:13:10.424647Z'
     assert detection_3['created_timestamp'] == '2023-04-20T11:13:10.424Z'
+
+
+def test_enrich_groups__no_resources(mocker):
```
Comment (ShahafBenYakir) [inline Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon_test.py:7528]:
```suggestion
def test_enrich_groups_no_resources(mocker):
```

**Review by ShahafBenYakir — COMMENTED** [review]
Nice
1. Did we check that it actually solves the customer issue?
2. Did we check that the UT fails without your fix?

**Comment by ilappe** [comment]
> Nice
> 
> 1. Did we check that it actually solves the customer issue?
> 2. Did we check that the UT fails without your fix?

1. yes
2. see the UT screenshots

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**